### DISCUSSION TITLE
Possibility of Microsoft 365 Certification

### DISCUSSION THREAD

**Discussion by nizhf** [abraunegg/onedrive#2480] [Q&A] [answered]
Hi,
I have used your OneDrive Linux client to sync files between my Windows and Ubuntu PCs. But recently, my university IT manager blocked all third-party Apps from logging into the university Microsoft 365 account because of "security issues." They told me they can only accept software officially proven by Microsoft: https://learn.microsoft.com/en-us/microsoft-365-app-certification/overview. Even though I told them this client is proven by so many users, they still did not want to change their decision. Is it feasible for you to obtain such a certification? Thank you.

**Comment by abraunegg** [comment, marked as answer]
@nizhf 
Publisher Verification has been configured (as per https://learn.microsoft.com/en-us/azure/active-directory/develop/publisher-verification-overview), however a Microsoft Bug with their verification process prevents the information that has been configured, to be verified.

For further reading, please read:
* https://github.com/MicrosoftDocs/azure-docs/issues/39665
* https://learn.microsoft.com/en-us/answers/questions/13291/why-does-azure-domain-verification-not-follow-the

The site that is configured within Azure (where the application id is specified) is set to https://abraunegg.github.io/

The required file is at the required location: https://abraunegg.github.io/.well-known/microsoft-identity-association.json

The Microsoft bug prevents the Publisher Verification from 'working' - this is a bug that Microsoft need to fix.

In terms of Microsoft 365 Certification - I can look into this further - however on my reading this is not going to be a simple process or even applicable as this application does not integrate with those provided applications.

> But recently, my university IT manager blocked all third-party Apps from logging into the university Microsoft 365 account because of "security issues." 

More than happy to have a pragmatic discussion with your University IT Manager to understand their concern and potentially give them the correct information to potentially shift their position. 
Additionally, they should have a read of this: https://github.com/abraunegg/onedrive/blob/master/docs/application-security.md
